exampleWidget/widget.ts

```diff
@@ -40,6 +40,26 @@ class StatefulMarqueeWidget extends HTMLElement {
     this.shadowRoot?.appendChild(template.content.cloneNode(true))
   }
 }
+class StatefulMarqueeWidget2 extends HTMLElement {
+  static get is () {
+    return 'stateful-marquee-widget-2'
+  }
+
+  constructor () {
+    super()
+    this.attachShadow({ mode: 'open' })
+    client.on('counter', (cnt) => {
+      this.shadowRoot!.querySelector('div')!.innerHTML = (
+        'Hello World' +
+        [...new Array(cnt)].map(() => '!').join('')
+      )
+    })
+  }
+
+  connectedCallback () {
+    this.shadowRoot?.appendChild(template.content.cloneNode(true))
+  }
+}
```

Comment:
i think it makes sense to showcase two example per extension in this example but let's be a bit creative when it comes to how this other widget is called and what it does 😉 here are some suggestions:

- change the name of the custom element
- maybe show case tangle states rather than just a counter?

Reply:
Yeah I can come up with something then